build.gradle

```diff
@@ -19,6 +19,12 @@ repositories {
     jcenter()
 }
 
+bootJar {
```

Comment:
Doesn't `bootJar` inherit from `jar` anyway?

Reply:
It didn't work for me at the time

Reply:
I've tested again with the new gradle and doesn't work either. Has to be bootJar for some magical reason.